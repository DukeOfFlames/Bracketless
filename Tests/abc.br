START

Â°x = 5

fn square{x:int}{
    -> {x!!?? - x} ^ 2
}

#drucke{square{x}}

END