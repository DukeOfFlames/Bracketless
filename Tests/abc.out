[Float:
  7.888609052210118e-31]
